### PR TITLE
INJICERT-213- Negative testcases

### PR DESCRIPTION
**Pull request by likhitharl**
Negative testcases for Sunbird UseCase and added description -unique identifier for each testcase

### CODE REVIEW

File: apitest/src/main/resources/injicertify/SunBirdCNegative/AuthenticateUserSunBirdCNeg/AuthenticateUserSunBirdCNeg.hbs [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+	"encodedHash": "{{encodedHash}}",
```
Comment (mohanachandran-s) [inline apitest/src/main/resources/injicertify/SunBirdCNegative/AuthenticateUserSunBirdCNeg/AuthenticateUserSunBirdCNeg.hbs:2]: Correct the indentation

File: apitest/src/main/resources/injicertify/SunBirdCNegative/AuthorizationCodeSunBirdCNeg/AuthorizationCodeSunBirdCNeg.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+AuthorizationCodeSunBirdC:
+   InjiCertify_ESignet_AuthorizationCode_SunBirdC_All_Valid_Smoke_sid:
```
Comment (mohanachandran-s) [inline apitest/src/main/resources/injicertify/SunBirdCNegative/AuthorizationCodeSunBirdCNeg/AuthorizationCodeSunBirdCNeg.yml:2]: is this testcase name is unique?

File: apitest/src/main/resources/injicertify/SunBirdCNegative/GetCredentialSunBirdCNeg/GetCredentialSunBirdCNeg.hbs [hunk 3]
```diff
@@ -0,0 +1,23 @@
+{
+	"client_id": "{{client_id}}",
+	"idpAccessToken": "{{idpAccessToken}}",
```
Comment (mohanachandran-s) [inline apitest/src/main/resources/injicertify/SunBirdCNegative/GetCredentialSunBirdCNeg/GetCredentialSunBirdCNeg.hbs:3]: Correct the indentation

File: apitest/src/main/resources/injicertify/SunBirdCNegative/GetCredentialSunBirdCNeg/GetCredentialSunBirdCNeg.yml [hunk 4]
```diff
@@ -0,0 +1,808 @@
...
+        "format": "ldp_vc",
+      	"type": [{types: "VerifiableCredential"}],
+      	"@context": [{context: "$VCICONTEXTURL$"}],
+      	"proof_type": "jwt",
+        "proof_jwt": "$PROOF_JWT_2$"
+}'
+      output: '{
+      "error": "unsupported_credential_type"
+}'
+   InjiCertify_GetCredentialSunBirdC_IdpAccessToken_onlyone_credentialtype_2:
```
Comment (mohanachandran-s) [inline apitest/src/main/resources/injicertify/SunBirdCNegative/GetCredentialSunBirdCNeg/GetCredentialSunBirdCNeg.yml:111]: is these are negative testcases, if yes please mention it in testcase name

File: apitest/src/main/resources/injicertify/SunBirdCNegative/OAuthDetailsRequestSunBirdCNeg/OAuthDetailsRequestSunBirdCNeg.yml [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+      	"scope": "sunbird_rc_insurance_vc_ldp",
+      	"responseType": "code",
+      	"redirectUri": "$IDPREDIRECTURI$",
+      	"display": "popup",
+	    "prompt": "login",
+	    "acrValues": "mosip:idp:acr:knowledge",
+	    "nonce": "973eieljzng",
+	    "state": "eree2311",
+	    "claimsLocales": "en",
+	    "codeChallenge": "$CODECHALLENGE$",
```
Comment (mohanachandran-s) [inline apitest/src/main/resources/injicertify/SunBirdCNegative/OAuthDetailsRequestSunBirdCNeg/OAuthDetailsRequestSunBirdCNeg.yml:21]: Correct the indentation

File: apitest/src/main/resources/injicertify/SunBirdCNegative/AuthorizationCodeSunBirdCNeg/AuthorizationCodeSunBirdCNeg.yml (hunk 2 above)
Comment (mohanachandran-s) [reply apitest/src/main/resources/injicertify/SunBirdCNegative/AuthorizationCodeSunBirdCNeg/AuthorizationCodeSunBirdCNeg.yml:2]: Description and unique identifier missing

File: apitest/src/main/resources/injicertify/SunBirdCNegative/GetCredentialSunBirdCNeg/GetCredentialSunBirdCNeg.yml [hunk 6]
```diff
@@ -0,0 +1,808 @@
+GetCredentialSunBirdC:
+   InjiCertify_GetCredentialSunBirdC_IdpAccessToken_invalid_format:
+      endPoint: $INJICERTIFYINSURANCEBASEURL$/v1/certify/issuance/credential
+      uniqueIdentifier: TC_injicertify_credentialissuance_01      
+      description: Get sunbird VC when invalid format is given      
+      role: resident
+      checkErrorsOnlyInResponse: true
+      restMethod: post
+      validityCheckRequired: true
+      inputTemplate: injicertify/SunBirdC/GetCredentialSunBirdC/GetCredentialSunBirdC
```
Comment (mohanachandran-s) [inline apitest/src/main/resources/injicertify/SunBirdCNegative/GetCredentialSunBirdCNeg/GetCredentialSunBirdCNeg.yml:10]: input template is pointing to different folder

File: apitest/src/main/resources/injicertify/SunBirdCNegative/GenerateTokenSunBirdCNeg/GenerateTokenSunBirdCNeg.yml [hunk 7]
```diff
@@ -0,0 +1,20 @@
+GenerateTokenSunBirdC:
```
Comment (mohanachandran-s) [inline apitest/src/main/resources/injicertify/SunBirdCNegative/GenerateTokenSunBirdCNeg/GenerateTokenSunBirdCNeg.yml:1]: this has to be "GenerateTokenSunBirdCNeg"

**Review by mohanachandran-s — CHANGES_REQUESTED** [review]

File: apitest/src/main/resources/injicertify/SunBirdCNegative/AuthorizationCodeSunBirdCNeg/AuthorizationCodeSunBirdCNeg.yml (hunk 2 above)
Comment (likhitharl) [reply apitest/src/main/resources/injicertify/SunBirdCNegative/AuthorizationCodeSunBirdCNeg/AuthorizationCodeSunBirdCNeg.yml:2]: Updated the changes

File: apitest/src/main/resources/injicertify/SunBirdCNegative/GetCredentialSunBirdCNeg/GetCredentialSunBirdCNeg.yml (hunk 6 above)
Comment (likhitharl) [reply apitest/src/main/resources/injicertify/SunBirdCNegative/GetCredentialSunBirdCNeg/GetCredentialSunBirdCNeg.yml:10]: Updated

File: apitest/src/main/resources/injicertify/SunBirdCNegative/GenerateTokenSunBirdCNeg/GenerateTokenSunBirdCNeg.yml (hunk 7 above)
Comment (likhitharl) [reply apitest/src/main/resources/injicertify/SunBirdCNegative/GenerateTokenSunBirdCNeg/GenerateTokenSunBirdCNeg.yml:1]: Updated

File: apitest/src/main/resources/injicertify/SunBirdCNegative/GetCredentialSunBirdCNeg/GetCredentialSunBirdCNeg.yml (hunk 4 above)
Comment (likhitharl) [reply apitest/src/main/resources/injicertify/SunBirdCNegative/GetCredentialSunBirdCNeg/GetCredentialSunBirdCNeg.yml:111]: Updated

File: apitest/src/main/resources/injicertify/SunBirdCNegative/OAuthDetailsRequestSunBirdCNeg/OAuthDetailsRequestSunBirdCNeg.yml (hunk 5 above)
Comment (likhitharl) [reply apitest/src/main/resources/injicertify/SunBirdCNegative/OAuthDetailsRequestSunBirdCNeg/OAuthDetailsRequestSunBirdCNeg.yml:23]: updated

File: apitest/src/main/resources/injicertify/SunBirdCNegative/GetCredentialSunBirdCNeg/GetCredentialSunBirdCNeg.hbs (hunk 3 above)
Comment (likhitharl) [reply apitest/src/main/resources/injicertify/SunBirdCNegative/GetCredentialSunBirdCNeg/GetCredentialSunBirdCNeg.hbs:3]: updated

File: apitest/src/main/resources/injicertify/SunBirdCNegative/AuthenticateUserSunBirdCNeg/AuthenticateUserSunBirdCNeg.hbs (hunk 1 above)
Comment (likhitharl) [reply apitest/src/main/resources/injicertify/SunBirdCNegative/AuthenticateUserSunBirdCNeg/AuthenticateUserSunBirdCNeg.hbs:2]: updated

**Review by mohanachandran-s — APPROVED** [review]

**Pull request merged**